src/components/Login.js

```diff
@@ -44,27 +43,11 @@ const Login = () => {
       if (user) {
         setUser(user);
         setCurrentUsername(user.username);
-        handleMyOrder();
         history.push("/authenticated");
       }
     });
   };
 
-  const handleMyOrder = () => {
-    console.log("show me inside handleMyOrder");
-    getOrderByUser()
-      .then((r) => {
-        console.log("show me rrrr inside handleMyOrder", r);
-        setMyOrder(r);
-        setTotal(() => {
-          return r.products.reduce((acc, p) => {
-            return acc + p.quantity * p.price;
-          }, 0);
-        });
-      })
-      .catch((e) => console.error(e));
-  };
-
```

Comment:
I deleted handleMyOrder in Login.js because the useEffect that does the same thing in index.js would be executed every time currentUsername changes.